From 346ca78aa710c0081af08838d5f31fc5a76065c2 Mon Sep 17 00:00:00 2001
From: Aleksander Morgado <aleksander@aleksander.es>
Date: Sun, 23 May 2021 14:17:57 +0200
Subject: [PATCH 16/39] iface-modem-3gpp: remove redundant log message

It doesn't give us any valuable information, so just remove it.
---
 src/mm-iface-modem-3gpp.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/mm-iface-modem-3gpp.c b/src/mm-iface-modem-3gpp.c
index f0e9fe31..cc39fa08 100644
--- a/src/mm-iface-modem-3gpp.c
+++ b/src/mm-iface-modem-3gpp.c
@@ -2374,7 +2374,6 @@ sync_eps_bearer_ready (MMIfaceModem3gpp *self,
     g_autoptr(MMBearerProperties)  properties  = NULL;
     g_autoptr(GError)              error = NULL;
 
-    mm_obj_dbg (self, "EPS bearer sync ready");
     ctx = g_task_get_task_data (task);
 
     properties = MM_IFACE_MODEM_3GPP_GET_INTERFACE (self)->load_initial_eps_bearer_finish (self, res, &error);
-- 
2.31.1

